### DISCUSSION TITLE
Pi zero W to use USB audio, "deactivate" HDMI and strange behaviour of shairport-sync

### DISCUSSION THREAD

**Discussion by bamboleate** [mikebrady/shairport-sync#1834] [Q&A]
I just updated my last Raspberry Pi Zero from Buster to Bullseye along with that became a reinstall of the "classic" Shairport Sync, and I'd like to share some of the experiences:

1. I seem to have some strange version of a Pi Zero that causes kernel oopses when running the first commands of [this manual](https://github.com/mikebrady/shairport-sync/blob/master/BUILD.md), namely this command:
   ```
   apt-get install --no-install-recommends build-essential git autoconf automake libtool \
    libpopt-dev libconfig-dev libasound2-dev avahi-daemon libavahi-client-dev libssl-dev libsoxr-dev
   ```
   I have to edit `/boot/config.txt` and add/edit the line:
   ```
   over_voltage=2
   ```
   to not get the kernel oopses in Bullseye.
2. I have a Pi Zero W, and when I install Shairport Sync as is, so without any fiddling with the many options, the default output is HDMI, which I didn't want to use, since I have a USB soundcard that I do want to use.
   Here's the issue: I could select the USB device via `alsamixer` and when I start Shairport Sync via:
   ```
   $ shairport-sync
   ```
   it runs and I can play audio through the USB device. HOWEVER, if I start Shairport Sync via:
   ```
   # systemctl start shairport-sync
   ```
   NO audio is played and I assume that HDMI is selected automatically, as it is the default device.
   
   This is why I'm writing this, along with maybe helping other people with the same/similar issues.

   There sure is a working solution in `shairport-sync.conf`, but I found this to be fiddly, as I ALWAYS have to try several times to find out, what to write there. 

3. So here is what I found: [https://www.alsa-project.org/wiki/Setting_the_default_device](https://www.alsa-project.org/wiki/Setting_the_default_device)
   Just follow the short explanation on what to do and enjoy  ;)

Thank you @mikebrady for this marvellous piece of software!

PS feel free to edit or delete

**Comment by mikebrady** [comment]
Thanks for the post and for the kind words. I've taken you at your word and edited it for formatting and a few small things -- I hope it's okay.
